2021-08-26 22:39:24,410 INFO    MainThread:56206 [wandb_setup.py:_flush():69] setting env: {'entity': 'rishanthrajendhran', 'project': 'eigengame', 'sweep_id': '4yfv7oi2', 'root_dir': '/Users/rishanthrajendhran/Desktop/EigenGame', 'run_id': '3xh1g6yo', 'sweep_param_path': '/Users/rishanthrajendhran/Desktop/EigenGame/wandb/sweep-4yfv7oi2/config-3xh1g6yo.yaml'}
2021-08-26 22:39:24,410 INFO    MainThread:56206 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-26 22:39:24,410 INFO    MainThread:56206 [wandb_init.py:_log_setup():336] Logging user logs to /Users/rishanthrajendhran/Desktop/EigenGame/wandb/run-20210826_223924-3xh1g6yo/logs/debug.log
2021-08-26 22:39:24,410 INFO    MainThread:56206 [wandb_init.py:_log_setup():337] Logging internal logs to /Users/rishanthrajendhran/Desktop/EigenGame/wandb/run-20210826_223924-3xh1g6yo/logs/debug-internal.log
2021-08-26 22:39:24,410 INFO    MainThread:56206 [wandb_init.py:init():369] calling init triggers
2021-08-26 22:39:24,410 INFO    MainThread:56206 [wandb_init.py:init():376] wandb.init called with sweep_config: {'learningRate': 0.01}
config: {'xDim': (100000, 4), 'numIterations': 10000, 'k': 4, 'learningRate': 1}
2021-08-26 22:39:24,410 INFO    MainThread:56206 [wandb_init.py:init():418] starting backend
2021-08-26 22:39:24,411 INFO    MainThread:56206 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-26 22:39:24,435 INFO    MainThread:56206 [backend.py:ensure_launched():132] starting backend process...
2021-08-26 22:39:24,441 INFO    MainThread:56206 [backend.py:ensure_launched():137] started backend process with pid: 56216
2021-08-26 22:39:24,444 INFO    MainThread:56206 [wandb_init.py:init():423] backend started and connected
2021-08-26 22:39:24,445 INFO    MainThread:56206 [wandb_run.py:_config_callback():787] config_cb None None {'learningRate': 0.01}
2021-08-26 22:39:24,446 INFO    MainThread:56206 [wandb_init.py:init():465] updated telemetry
2021-08-26 22:39:24,447 INFO    MainThread:56206 [wandb_init.py:init():484] communicating current version
2021-08-26 22:39:25,452 INFO    MainThread:56206 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.12.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-08-26 22:39:25,453 INFO    MainThread:56206 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-08-26 22:39:25,838 INFO    MainThread:56206 [wandb_init.py:init():522] starting run threads in backend
2021-08-26 22:39:30,837 INFO    MainThread:56206 [wandb_run.py:_console_start():1565] atexit reg
2021-08-26 22:39:30,837 INFO    MainThread:56206 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-08-26 22:39:30,838 INFO    MainThread:56206 [wandb_run.py:_redirect():1417] Redirecting console.
2021-08-26 22:39:30,839 INFO    MainThread:56206 [wandb_run.py:_redirect():1479] Redirects installed.
2021-08-26 22:39:30,839 INFO    MainThread:56206 [wandb_init.py:init():546] run started, returning control to user process
2021-08-26 22:41:04,419 INFO    MainThread:56206 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 1
2021-08-26 22:41:04,422 INFO    MainThread:56206 [wandb_run.py:_restore():1507] restore
2021-08-26 22:41:06,435 INFO    MainThread:56206 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 776
  total_bytes: 776
}

2021-08-26 22:41:06,747 INFO    MainThread:56206 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 776
  total_bytes: 776
}

2021-08-26 22:41:07,110 INFO    MainThread:56206 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 776
  total_bytes: 15280
}

2021-08-26 22:41:07,692 INFO    MainThread:56206 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15280
  total_bytes: 15280
}

2021-08-26 22:41:07,799 INFO    MainThread:56206 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15280
  total_bytes: 15280
}

2021-08-26 22:41:07,910 INFO    MainThread:56206 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15280
  total_bytes: 15280
}

2021-08-26 22:41:08,017 INFO    MainThread:56206 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15280
  total_bytes: 15280
}

2021-08-26 22:41:08,125 INFO    MainThread:56206 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15280
  total_bytes: 15280
}

2021-08-26 22:41:09,550 INFO    MainThread:56206 [wandb_run.py:_show_files():1879] logging synced files
